### ISSUE TITLE
Hyperparameter Experiments

### ISSUE THREAD

**Issue by petermatts** [petermatts/QNNs#4]
Run files with different hyperparameter sets and compare performance.
This includes but is not limited to:
- Training parameters
- Model shape, i.e. number and size of layers
- Different PQC designs?

Document results #3

**Comment by petermatts** [comment]
I am closing as wontfix.

The MATLAB environment does not allow for modifying the PQC without needing to manually recompute the gradient. This will not scale moving forward.

Explore looking into TensorFlow or PyTorch with [PennyLane.ai](https://pennylane.ai/). Will open a new issue to develop this when the time arises. Making a note of this in the readme as well.
